src/main/scala/com/typesafe/sbt/packager/distroless/DistrolessPlugin.scala

```diff
@@ -0,0 +1,262 @@
+package com.typesafe.sbt.packager
+package distroless
+
+import com.typesafe.sbt.SbtNativePackager.Universal
+import com.typesafe.sbt.packager.Keys.{defaultLinuxInstallLocation, maintainer, scriptClasspath}
+import com.typesafe.sbt.packager.archetypes.scripts.*
+import com.typesafe.sbt.packager.docker.DockerPlugin.UnixSeparatorChar
+import com.typesafe.sbt.packager.docker.{Cmd, CmdLike, DockerPlugin, ExecCmd}
+import sbt.*
+import sbt.Keys.{discoveredMainClasses, mainClass, mappings, streams}
+
+/**
+  * ==Distroless Plugin==
+  *
+  * This plugin helps you build docker containers with distroless images
+  *
+  * ==Configuration==
+  *
+  * In order to configure this plugin take a look at the available [[com.typesafe.sbt.packager.docker.DockerKeys]]
+  *
+  * ==Requirements==
+  *
+  * You need docker to have docker installed on your system and be able to execute commands. Check with a single
+  * command:
+  *
+  * {{{
+  * docker version
+  * }}}
+  *
+  * @example
+  *   Enable the plugin in the `build.sbt`
+  *   {{{
+  *    enablePlugins(DistrolessPlugin)
+  *   }}}
+  */
+object DistrolessPlugin extends AutoPlugin {
+
+  override def requires: Plugins = DockerPlugin
+
+  object autoImport extends Keys
+
+  import DistrolessPlugin.autoImport.*
+  import DockerPlugin.autoImport.*
+
+  override def projectConfigurations: Seq[Configuration] = Seq()
+
+  // Some of the default values are now provided in the global setting based on
+  // sbt plugin best practice: https://www.scala-sbt.org/release/docs/Plugins-Best-Practices.html#Provide+default+values+in
+  override lazy val globalSettings: Seq[Setting[?]] =
+    Seq(
+      dockerBaseImage := "gcr.io/distroless/java",
+      dockerEntrypoint := Seq("/usr/bin/java"),
```

Comment:
Initially I didn't want to override the entrypoint defined by the image, but by default is `java -jar`, expecting a single jar as CMD. This limits its use to simple JARs with no libraries, or to FatJAR/UberJARs